### PR TITLE
Add MySQL Schema Manager Docs

### PR DESCRIPTION
**Pull request by DRoyyy**
Signed-off-by: DRoyyy <diptaroy@appscode.com>

### CODE REVIEW

File: docs/guides/mysql/concepts/mysqldatabase/index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# MySQLDatabase
+
+## MySQLDatabase Specification
+
+As with all other Kubernetes objects, an `MySQLDatabase` needs `apiVersion`, `kind`, and `metadata` fields. It also needs a `spec` section.
```
Comment (heheh13) [inline docs/guides/mysql/concepts/mysqldatabase/index.md:22]: briefly describe what `mysqldatabase` is.

File: docs/guides/mysql/concepts/mysqldatabase/index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    parent: guides-mysql-concepts
+    weight: 30
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+# MySQLDatabase
+
```
Comment (TasdidurRahman) [inline docs/guides/mysql/concepts/mysqldatabase/index.md:16]: add this : 
What is MySQLDatabase ? 
MySQLDatabase is a Kubernetes Custom Resource Definitions (CRD). It provides  a declarative way of implementing multitenancy inside KubeDB provisioned MySQL server. You need to describe the target database, desired database configuration, the vault server reference for managing the user in a MySQLDatabase object, and the KubeDB Schema Manager operator will create Kubernetes objects in the desired state for you.

File: docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+  allowedSchemas:
+    namespaces:
+      from: Selector
+      selector:
+        matchLabels:
+          app: schemaManager
+  terminationPolicy: WipeOut
+```
+
+In this yaml,
```
Comment (heheh13) [inline docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md]: here

File: docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+        resources:
+          requests:
+            storage: 1Gi
+  authMethods:
+    - type: kubernetes
+      path: kubernetes
+  terminationPolicy: WipeOut
+```
+
+In this yaml,
```
Comment (heheh13) [inline docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md]: here

File: docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md [hunk 5]
```diff
@@ -0,0 +1,428 @@
...
+    namespace: demo
+  accessPolicy:
+    subjects:
+      - kind: ServiceAccount
+        name: "tester"
+        namespace: "demox"
+    defaultTTL: "5m"
+  deletionPolicy: "Delete"
+```
+In this yaml,
```
Comment (heheh13) [inline docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md]: Here,

File: docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md [hunk 6]
```diff
@@ -0,0 +1,428 @@
...
+
+> **Note:** YAML files used in this tutorial are stored in [docs/guides/mysql/schema-manager/deploy-mysqldatabase/yamls](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/guides/mysql/schema-manager/deploy-mysqldatabase/yamls) directory of [kubedb/doc](https://github.com/kubedb/docs) repository.
+
+## Deploy MySQL Server and Vault Server 
+
+Firstly, we are going to deploy a `MySQL Server` by using `KubeDB` operator. Also, we are deploying a `Vault Server` using `KubeVault` Operator.
+
+### Deploy MySQL Server
+
+In this section, we are going to deploy a MySQL Server. Below is the YAML of the `MySQL` CR that we are going to create,
```
Comment (heheh13) [inline docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md]: `let's deploy a MySQL server using the following yaml`

Comment (heheh13) [reply docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md]: or 
`let's deploy a MySQL server using the following yaml. which will create a MySQL standalone instance of MySQL version 8.0.29`

**Review by heheh13 — COMMENTED** [review]
try to remove `Below is the YAML of the `Namespace` that we are going to create,` this type of statement.
instead try to replace with `lets create the namespace we want using this yaml `

**Review by heheh13 — DISMISSED** [review]
LGTM!

File: docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md [hunk 7]
```diff
@@ -0,0 +1,428 @@
...
+mysql> SHOW CREATE DATABASE demo_user;
++-----------+-------------------------------------------------------------------------------------------------------+
+| Database  | Create Database                                                                                       |
++-----------+-------------------------------------------------------------------------------------------------------+
+| demo_user | CREATE DATABASE `demo_user` /*!40100 DEFAULT CHARACTER SET big5 */ /*!80016 DEFAULT ENCRYPTION='N' */ |
++-----------+-------------------------------------------------------------------------------------------------------+
+1 row in set (0.00 sec)
+```
+Here, we can see that the `spec.database.config.characterSet` is changed to `big5`. So, our database altering has been successful. 
+
```
Comment (TasdidurRahman) [inline docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md:411]: describe what happens when the MySQLDatabase is deleted

File: docs/guides/mysql/schema-manager/initializing-with-script/index.md [hunk 8]
```diff
@@ -0,0 +1,346 @@
...
+```bash
+$ kubectl get mysqldatabase schema-script -n demox -o=jsonpath='{.status.authSecret.name}'
+schema-script-mysql-req-s85fuw
+
+$ kubectl view-secret schema-script-mysql-req-s85fuw -n demox -a
+password=DueiiR-JyGpa3rejG2Zd
+username=v-kubernetes-k8s.dc833e-yb9r7uhs
+```
+
+### Insert Sample Data
```
Comment (TasdidurRahman) [inline docs/guides/mysql/schema-manager/initializing-with-script/index.md]: fix the title : check initialization

**Review by TasdidurRahman — CHANGES_REQUESTED** [review]

**Review by TasdidurRahman — DISMISSED** [review]
LGTM

**Review by heheh13 — DISMISSED** [review]

**Pull request merged**